src/components.typ

```diff
@@ -113,7 +113,7 @@
 /// That means that `#checkerboard[...][...]` stacks horizontally and `#checkerboard(columns: 1)[...][...]` stacks vertically.
```

Comment:
The new parameters should be explained in the comments

Reply:
Done :slightly_smiling_face: